lib/confirm-controller.js

```diff
@@ -1,61 +1,129 @@
 'use strict';
 
-const BaseController = require('./base-controller');
 const _ = require('lodash');
+const i18nLookup = require('i18n-lookup');
+const Mustache = require('mustache');
```

Comment:
@joefitter Aside from functionality, there is a benefit in managing fewer dependencies. If we can use only one of the two then we probably should